disnake/guild.py

```diff
@@ -26,7 +26,7 @@
 from . import abc, utils
 from .app_commands import GuildApplicationCommandPermissions
 from .asset import Asset
-from .automod import AutoModRule
+from .automod import AutoModAction, AutoModRule, AutoModTriggerMetadata
```

Comment:
Minor nitpick, `AutoModTriggerMetadata` doesn't appear to be needed outside the `TYPE_CHECKING` block unless I misread

Reply:
Yup, fair point. Fixed in https://github.com/DisnakeDev/disnake/pull/1030/commits/88f0b02e2ab02f4a363b152deed54e1b89cc32bf